collections/corine-land-cover-accounting-layers.yaml

```diff
@@ -67,6 +67,11 @@ Configurations:
     sentinelhub:layer_name: "Corine Land Cover Accounting Layer"
     sentinelhub:mosaicking_order: "mostRecent"
     sentinelhub:upsampling: "BICUBIC"
+DocumentationLinks:
+  - href: https://collections.eurodatacube.com/corine-land-cover-accounting-layers/readme.html#band-information
```

Comment:
Can you please put all the strings here into quotations (see how it's for the objects under configurations for reference). Same for the corine-land-cover collection.